WcaOnRails/.rubocop_todo.yml

```diff
@@ -234,6 +234,7 @@ Style/ClassAndModuleChildren:
     - 'app/controllers/accounts/registrations_controller.rb'
     - 'app/controllers/api/v0/api_controller.rb'
     - 'app/controllers/api/v0/competitions_controller.rb'
+    - 'app/controllers/api/v0/records_controller.rb'
```

Comment:
Dang, that's confusing. I agree with the author: "I prefer to avoid thinking where possible, so I try to stay away from the module A::B syntax". @coder13, let's listen to rubocop here.